app/server.py

```diff
@@ -35,6 +37,18 @@ def learnToInvest(request):
     page = app.templates.get_template('resources.html')
     return page.render()
 
+@app.route('/api/v1/saveohlvc', methods=['POST'])
+def saveOhlvc(request):
+    """ get and save all recent daily stock data for a symbol """
+    import db
```

Comment:
no reason really.  I didn't read the pep8 about importing module and I thought if I was only using this import in one method then put it there, but I'm not sure why I thought that.  Thanks for the knowledge